charts/k8ssandra/templates/stargate/cassandra-ingress.yaml

```diff
@@ -1,14 +1,14 @@
-{{- $traefik := .Values.ingress.traefik -}}
-{{- if $traefik.enabled }}
+{{- $stargateIngress := .Values.stargate.ingress -}}
+{{- if $stargateIngress.enabled }}
 {{- $releaseName := .Release.Name -}}
 {{- $datacenterName := include "k8ssandra.datacenterName" . -}}
 
-{{- if $traefik.stargate.enabled }}
-{{- if and $traefik.stargate.cassandra.enabled $traefik.cassandra.enabled }}
-  {{- fail "ingress.traefik.stargate.cassandra.enabled and ingress.traefik.cassandra.enabled cannot both be enabled." }}
+{{- if and $stargateIngress.cassandra.enabled .Values.ingress.traefik.cassandra.enabled }}
```

Comment:
Is it safe to assume `.Values.ingress.traefik.cassandra.enabled` will be something like `$cassandra.ingress.enabled` as part of a separate ticket?

Reply:
> Reaper, and Non-Stargate Cassandra will come in later commits to this PR.

I'm going to close this question since you already answered it in the PR description.

Reply:
Yes, part of this PR, coming in later commits.